contributed_definitions/NXcxi_ptycho.nxdl.xml

```diff
@@ -59,15 +59,13 @@
     </symbols>
 
     <doc>
-          Application definition for a ptychography experiment. This is compatible with CXI from version 1.6 (discussed with Filippe Maia) if this application definition
+          Application definition for a ptychography experiment. This is compatible with CXI from version 1.6 if this application definition
```

Comment:
This should be a complete sentence on only one line (used as a summary of this NXDL), to be consistent with the other NXDL files.  As it stands now, this image shows how the current summary will appear:

![Clipboard01](https://user-images.githubusercontent.com/2279984/66685352-df9abc00-ec41-11e9-941b-7ffd39521824.jpg)

Compare with current NeXus documentation: http://download.nexusformat.org/doc/html/classes/contributed_definitions/index.html#contributed-definitions

Reply:
*complete sentence* - not so much it seems

possibly:
*definition for a ptychography experiment, compatible with CXI from version 1.6*